CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 cmake_minimum_required(VERSION 3.16)
 
 # Change project name, based on EPICS version
+OPTION(EPICS_VERSION "Select major version of EPICS to build (3 or 7)")
+
+if(NOT EPICS_VERSION)
```

Comment:
I find the naming with EPICS_VERSION and EPICS_VER a bit confusing and unfortunate and mildly confusing

Reply:
I eliminated EPICS_VER now completely